cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -1784,7 +1784,7 @@ Slice::Python::CodeVisitor::writeHash(const string& name, const TypePtr& p, int&
         return;
     }
 
-    _out << nl << "_h = 5 * _h + Ice.getHash(" << name << ")";
+    _out << nl << "_h = 5 * _h + _builtins.hash(" << name << ")";
```

Comment:
no need to use a wrapper function to call the `builtins.hash` methods, we can use an import alias.